app/index.html

```diff
@@ -52,6 +52,7 @@
                           <a href="" class="dropdown-toggle" data-toggle="dropdown"><i class="glyphicon glyphicon-user"></i><span ng-bind="sessionUser"></span><b class="caret"></b></a>
                           <ul class="dropdown-menu animated fadeInUp">
                             <li><a href="#/profile">Profile</a></li>
+                            <li><a href="#/about">About</a></li>
```

Comment:
If a non-admin user ins't allowed to see the the system version, im not sure if this should be visible to them then it its wont return anything to them. Also, should we not allow a normal user to see the system version?